Gems/PhysX/Code/Source/RigidBodyComponent.cpp

```diff
@@ -259,10 +267,12 @@ namespace PhysX
         {
             m_interpolator->SetTarget(transform.GetTranslation(), rigidBody->GetOrientation(), fixedDeltaTime);
         }
-        else
+        else if (AZ::TransformInterface* entityTransform = GetEntity()->GetTransform())
         {
-            AZ::TransformBus::Event(GetEntityId(), &AZ::TransformInterface::SetWorldRotationQuaternion, rigidBody->GetOrientation());
-            AZ::TransformBus::Event(GetEntityId(), &AZ::TransformInterface::SetWorldTranslation, rigidBody->GetPosition());
+            AZ::Transform newWorldTransform = entityTransform->GetWorldTM();
```

Comment:
Why not just use 'transform' which you already got from rigidBody->GetTransform() above?

Reply:
I was thinking about removing `m_enableActiveActors` entirely (i.e. have it always on) since our benchmarks showed no difference in performance even if every single object is moving. 
Seems this ability to switch it off creates more problems (having to support and test both modes) than it actually solves

re: transform
This is because entity transform can have scale but the rigid body one is always without the scale (i.e. only position + orientation)

Reply:
Did you test m_enableActiveActors = true and false with everything else being equal? I just want to verify that turning it on didn't have a performance cost that was just offset by the other part of the change where you now only have one call to SetWorldTM instead of two separate calls to SetWorldRotationQuaternion and SetWorldTranslation.

Otherwise, I agree it's probably best to just remove it so there's less to maintain.